public/static/docs/command-reference/get.md

```diff
@@ -65,6 +65,13 @@ name.
   the file or directory from. The tip of the default branch is used by default
   when this option is not specified.
 
+- `--show-url` - Returns path or url to the location in remote for given path,
```

Comment:
`Returns` -> `returns`

Reply:
why is it `path or ..`?

I would simplify significantly the whole piece.

"instead of downloading the file or directory print its actual location (URL) it would be downloaded from, e.g. s3://remote/storage/0c/121243234"

Reply:
Agree with Ivan. This could start:
```suggestion
- `--show-url` - returns URL to the remote location of the target data
```
but the rest of the paragraph can be simplified as well...

Reply:
@jorgeorpinel, is the distinction not important, that, it can also be a path, not just the URL?

Reply:
It's already confusing enough that the command takes both `url` and `path`, `url` accepting URLs and paths (explained in description), and then on top of that saying this option returns a "path or URL".
But OK I guess you're right, maybe check out the paragraph about the `url` command param and use similar language to explain this returned URL could be a local file system path (for local remotes).
OR don't use either path nor URL and just say something like "storage location" with URL corresponding to the type of remote, linking to remote cmd ref, where the "local" case is explained.

Reply:
@jorgeorpinel @skshetry what about the way I suggested it? One sentence, using terms users would understood? `instead of downloading the file or directory print its actual location (URL) it would be downloaded from, e.g. s3://remote/storage/0c/121243234`? One simple sentence + even example to put more context.

Reply:
Yeah I think you got it right Ivan. I added to the core repo in https://github.com/iterative/dvc/pull/3220/commits/ee0193ccd36fdc067ace131958f7960819202643 and will update here as well.

Reply:
Updated in 072defa3.